examples/street_canyon_radiosity.ipynb

```diff
@@ -0,0 +1,523 @@
+      "text/plain": [
+       "<Figure size 640x480 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Plot the simple BRDF with one value equal to every direction\n",
```

Comment:
I would suggest to use the [baloon plot](https://spharpy.readthedocs.io/en/stable/modules/spharpy.plot.html#spharpy.plot.balloon) instead, we also need to show the incident direction, maybe by a line, otherwise it is not really telling anything.